lntest/btcd.go

```diff
@@ -0,0 +1,87 @@
+package lntest
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/integration/rpctest"
+	"github.com/btcsuite/btcd/rpcclient"
+)
+
+// logDir is the name of the temporary log directory.
+var logDir = "./.backendlogs"
+
+// BtcdBackendConfig is an implementation of the BackendConfig interface
+// backed by a btcd node.
+type BtcdBackendConfig struct {
+	// rpcConfig houses the connection config to the backing btcd instance.
+	rpcConfig rpcclient.ConnConfig
+
+	// harness is the backing btcd instance.
+	harness *rpctest.Harness
```

Comment:
Nit: any reason we don't just store the `P2PAddr` itself rather than going through the `harness`?

Reply:
Nope, changed to just storing the addr!